pom.xml

```
@@ -60,15 +60,82 @@
             <artifactId>liquibase-core</artifactId>
             <version>4.27.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.opensearch.client</groupId>
+            <artifactId>opensearch-java</artifactId>
+            <version>2.10.2</version>
```

Couple of bugfixes included:

```suggestion
            <version>2.10.3</version>
```

---

We could also exclude `opensearch-rest-client` since we use  AHC 5 here (https://github.com/liquibase/liquibase-opensearch/pull/25/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R91)

```
            <exclusions>
            	<exclusion>
            		<groupId>org.opensearch.client</groupId>
            		<artifactId>opensearch-rest-client</artifactId>
            	</exclusion>
            </exclusions>

```